pinot-transport/src/main/java/com/linkedin/pinot/routing/RoutingTableSelector.java

```diff
@@ -26,7 +28,7 @@
    * @param realtimeTableName name of the realtime table (e.g. tableName_REALTIME)
    * @return
    */
-  public boolean shouldUseLLCRouting(final String realtimeTableName);
+  boolean shouldUseLLCRouting(final String realtimeTableName);
```

Comment:
Correct, there's a routing table creator that creates both hlc and llc routing tables and allows switching between both at query time.